static/css/components/form.olform.less

```diff
@@ -109,6 +109,8 @@
   // "You can search by author name (like j k rowling) or by Open Library ID (like OL23919A)."
   .tip {
     font-size: 11px;
+    display: inline-block;
+    min-width: 50px;
```

Comment:
Why do we need to specify this?

Reply:
So width, height and depth labels are aligned and of equal width